lx_administration/autoconf/imports/utils.py

```
@@ -91,12 +85,103 @@ def load_group_vars(group_vars_dir: Path) -> Dict[str, str]:
+
+
+def is_home_only_host(host):
+    """Returns True if host is ONLY for home configuration (not system)."""
+    return (
+        "home_config" in host.ansible_group_names and
+        not any(g for g in host.ansible_group_names if not g.startswith("home_") and g != "home_config")
+    )
+#efined home-only detection logic in utils .py
+#skips the export and YAML write for home-only hosts like abc@admin
+#Return home_only_hosts list
+#skips generating systems/x86_64-linux/<host>/default.nix.
```

Ruff (0.11.9)

No newline at end of file

Add trailing newline

(W292)